src/tree/SubscriptionTreeItem.ts

```diff
@@ -48,7 +48,19 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
     public async createChildImpl(context: ICreateChildImplContext): Promise<AzExtTreeItem> {
         const wizardContext: IStaticWebAppWizardContext = { accessToken: await getGitHubAccessToken(), client: createAzureClient(this.root, WebSiteManagementClient), ...context, ...this.root };
         const title: string = localize('createStaticApp', 'Create Static Web App');
-        const promptSteps: AzureWizardPromptStep<IStaticWebAppWizardContext>[] = [new StaticWebAppNameStep(), new GitHubOrgListStep(), new GitHubRepoListStep(), new GitHubBranchListStep(), new AppLocationStep(), new ApiLocationStep(), new AppArtifactLocationStep()];
+        const promptSteps: AzureWizardPromptStep<IStaticWebAppWizardContext>[] = [];
+
+        promptSteps.push(new StaticWebAppNameStep());
+        if (context.advancedCreation) {
```

Comment:
I feel like there's more changes necessary when advancedCreation is true. For example, you don't need to add `ResourceGroupCreateStep` or set `newResourceGroupName`